src/dashboard/install/README.md

```diff
@@ -150,28 +206,79 @@ This is the full list of environment strings supported:
+    - **Type:** `integer` 
+    - **Default:** `1`
 
-- [`dashboard.gunicorn-config.py`](./dashboard.gunicorn-config.py) -
-gunicorn configuration file sample.
+- **`AM_GUNICORN_WORKER_CLASS`**:
+    - **Description:** The type of worker processes to run.  See [WORKER-CLASS](http://docs.gunicorn.org/en/stable/settings.html#worker-class)
+    - **Type:** `string`
+    - **Default:** `gevent`
+
+- **`AM_GUNICORN_TIMEOUT`**:
+    - **Description:** Worker process timeout.  See [TIMEOUT](http://docs.gunicorn.org/en/stable/settings.html#timeout)
+    - **Type:** `integer` (seconds)
+    - **Default:** `172800`
 
-- [`dashboard.logging.json`](./dashboard.logging.json) - read the
-[logging configuration section](#logging-configuration) for more details.
+- **`AM_GUNICORN_RELOAD`**:
+    - **Description:** Restart workers when code changes.  See [RELOAD](http://docs.gunicorn.org/en/stable/settings.html#reload)
+    - **Type:** `boolean`
+    - **Default:** `false`
+
+- **`AM_GUNICORN_RELOAD_ENGINE`**:
+    - **Description:** Method of performing reload.  See [RELOAD-ENGINE](http://docs.gunicorn.org/en/stable/settings.html#reload-engine)
+    - **Type:** `string`
+    - **Default:** `auto`
+
+- **`AM_GUNICORN_CHDIR`**:
+    - **Description:** Directory to load apps from.  See [CHDIR](http://docs.gunicorn.org/en/stable/settings.html#chdir)
+    - **Type:** `string`
+    - **Default:** `/usr/share/archivematica/dashboard`
+
+- **`AM_GUNICORN_ACCESSLOG`**:
+    - **Description:** Location to write access log to.  See [ACCESSLOG](http://docs.gunicorn.org/en/stable/settings.html#accesslog)
+    - **Type:** `string`
+    - **Default:** `/dev/null`
+
+- **`AM_GUNICORN_ERRORLOG`**:
+    - **Description:** Location to write error log to.  See [ERRORLOG](http://docs.gunicorn.org/en/stable/settings.html#errorlog)
+    - **Type:** `string`
+    - **Default:** `-`
+
+- **`AM_GUNICORN_LOGLEVEL`**:
+    - **Description:** The granularity of Error log outputs.  See [LOGLEVEL](http://docs.gunicorn.org/en/stable/settings.html#loglevel)
+    - **Type:** `string`
+    - **Default:** `INFO`
+
+- **`AM_GUNICORN_PROC_NAME`**:
+    - **Description:** Name for this instance of gunicorn.  See [PROC-NAME](http://docs.gunicorn.org/en/stable/settings.html#proc-name)
+    - **Type:** `string`
+    - **Default:** `archivematica-dashboard`
 
 ## Logging configuration
 
-Archivematica 1.6.1 and earlier releases are configured by default to log to
-subdirectories in `/var/log/archivematica`, such as
-`/var/log/archivematica/dashboard/dashboard.debug.log`. Starting with
-Archivematica 1.7.0, logging configuration defaults to using stdout and stderr
+Archivematica 1.6.1 and earlier releases are configured by default to log to 
+subdirectories in /var/log/archivematica, such as 
+/var/log/archivematica/dashboard/dashboard.debug.log. Starting with 
+Archivematica 1.7.0, configuration defaults to using stdout and stderr
```

Comment:
In this change block, why do you delete the backticks?
Also you've added trailing whitespaces to three of the four lines.